### PR TITLE
Update flow to 0.241.0 to support aarch64 on linux

### PR DESCRIPTION
**Pull request by wendigo**
0.148.0 is a first version supporting aarch64 on Linux

## Release notes

( ) This is not user-visible or is docs only, and no release notes are required.
( ) Release notes are required. Please propose a release note for me.
( ) Release notes are required, with the following suggested text:

```markdown
# Section
* Fix some things. ({issue}`issuenumber`)
```

### CODE REVIEW

**Review by losipiuk — COMMENTED** [review]
"fix reported issue" is not really helpful as part of commit message

**Review by losipiuk — APPROVED** [review]

File: core/trino-main/src/main/resources/webapp/src/components/LivePlan.jsx [hunk 1]
```diff
@@ -314,13 +314,13 @@ export class LivePlan extends React.Component<LivePlanProps, LivePlanState> {
...
 
-    render() {
+    render() : any {
         const query = this.state.query;
 
         if (query === null || this.state.initialized === false) {
             let label = (<div className="loader">Loading...</div>);
             if (this.state.initialized) {
-                label = "Query not found";
+                label = <div className="error-message">Query not found</div>;
```
Comment (findepi) [inline core/trino-main/src/main/resources/webapp/src/components/LivePlan.jsx:323]: separate commit?

File: core/trino-main/src/main/resources/webapp/src/components/LivePlan.jsx [hunk 2]
```diff
@@ -181,7 +181,7 @@ export class LivePlan extends React.Component<LivePlanProps, LivePlanState> {
         }
     }
 
-    refreshLoop() {
+    refreshLoop = () => {
```
Comment (findepi) [inline core/trino-main/src/main/resources/webapp/src/components/LivePlan.jsx:184]: the added types and other changes like this one, are they strictly related to flow updated, or collateral?

**Review by findepi — APPROVED** [review]

File: core/trino-main/src/main/resources/webapp/src/components/LivePlan.jsx (hunk 2 above)
Comment (wendigo) [reply core/trino-main/src/main/resources/webapp/src/components/LivePlan.jsx:184]: related to flow update which validates if binding is correct

File: core/trino-main/src/main/resources/webapp/src/components/LivePlan.jsx (hunk 1 above)
Comment (wendigo) [reply core/trino-main/src/main/resources/webapp/src/components/LivePlan.jsx:323]: This is related to the flow update

**Pull request merged**